README.md

```diff
@@ -1,41 +1 @@
 # Altinn Platform
```

Comment:
Reading through this agin I think we could keep the products section. As the products.yaml is located at the root of the repo I think it should be mentioned in the README

```suggestion
# Altinn Platform

## Altinn Products
To configure or add identity federation (GitHub to Azure), Azure IAM, and handle Terraform state for a product in Altinn, modify the `products.yaml` file and create a pull request.
```